sql/core/src/main/scala/org/apache/spark/sql/jdbc/jdbc.scala

```diff
@@ -179,4 +184,63 @@ package object jdbc {
     }
 
   }
+
+  private [sql] case class DriverWrapper(wrapped: Driver) extends Driver {
```

Comment:
this doesn't need to be a case class, does it? Looks like you can just slightly change the pattern matching down below.